stdlib/3/email/message.pyi

```diff
@@ -36,16 +38,16 @@ class Message:
     def get_charset(self) -> _CharsetType: ...
     def __len__(self) -> int: ...
     def __contains__(self, name: str) -> bool: ...
-    def __getitem__(self, name: str) -> Optional[str]: ...
-    def __setitem__(self, name: str, val: str) -> None: ...
+    def __getitem__(self, name: str) -> Optional[_HeaderType]: ...
```

Comment:
For what it's worth, I think a Union return type is probably the right call here based on @daxtens's arguments.